client/extensions/woocommerce/app/dashboard/setup-task.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+
+// TODO add doc url and documentation link
+
+class SetupTask extends Component {
+	static propTypes = {
+		actions: PropTypes.arrayOf( PropTypes.shape( {
+			isSecondary: PropTypes.string.bool,
```

Comment:
Oy! Fixed in https://github.com/Automattic/wp-calypso/pull/14202/commits/66d13147f190def3689b815188fde46da04a36b4